pkgs/servers/etebase/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib
 , fetchFromGitHub
-, withLdap ? true
+, withLdap ? false
```

Comment:
I guess this was just accidentally pushed? Or do you propose changing the default here?

Reply:
Yes, I do - see commit message. I'll update the PR description in a moment :)

Reply:
This should probably be mentioned in the release notes. And we might as well remove the python module then.

Reply:
Ah right, i just removed the python package, wrote 2 sentences of release notes, squashed and rebased.